frontend/src/js/preview-v2/Preview.tsx

```diff
@@ -92,6 +93,16 @@ export default function Preview() {
     onClose();
   });
 
+  const store = useStore();
+  useEffect(() => {
+    if (!(store.getState() as StateT).panes.disableDragHandles) {
+      dispatch(toggleDragHandles());
+      return () => {
+        dispatch(toggleDragHandles());
+      };
+    }
+  });
```

Comment:
```suggestion
  }, []);
```
I think the effect only needs to be executed on component mount, without dependency array it would update on every rerender. Should be unnecessary imo.